From 59b80fe0eb1c9f5b511cd94215787e8bbec99389 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marek.vasut@nabladev.com>
Date: Sat, 23 Aug 2025 20:24:30 +0200
Subject: [PATCH 3/4] ARM: dts: stm32: Keep the reg11 and reg18 regulators
 always enabled

Do not disable reg11 and reg18, disabling these regulators causes
the SoC to hang.

Upstream-Status: Pending
Signed-off-by: Marek Vasut <marek.vasut@nabladev.com>
---
 arch/arm/dts/stm32mp13xx-dhcor-u-boot.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/dts/stm32mp13xx-dhcor-u-boot.dtsi b/arch/arm/dts/stm32mp13xx-dhcor-u-boot.dtsi
index 9ff42ab8248..7c8b9927b20 100644
--- a/arch/arm/dts/stm32mp13xx-dhcor-u-boot.dtsi
+++ b/arch/arm/dts/stm32mp13xx-dhcor-u-boot.dtsi
@@ -48,6 +48,14 @@
 	};
 };
 
+&reg11 {
+	regulator-always-on;
+};
+
+&reg18 {
+	regulator-always-on;
+};
+
 &sdmmc1 {
 	status = "disabled";
 };
-- 
2.50.1

